queue/src/fiskaltrust.Middleware.Storage.SQLite/SQLiteStorageBootstrapper.cs

```diff
@@ -49,6 +49,12 @@ public void ConfigureStorageServices(IServiceCollection serviceCollection)
             AddRepositories(serviceCollection);
         }
 
+        public async Task ConfigureStorageServicesAsync(IServiceCollection serviceCollection)
+        {
+            await InitAsync(_queueId, _configuration, _logger);
```

Comment:
Welp thats another thing. So I am not sure but IMO we should split up the initialization of our objects from the initialization of the database.